vulnfeeds/cmd/cvelist2osv/version_extraction.go

```diff
@@ -428,7 +425,6 @@ func buildVersionRange(intro string, lastAff string, fixed string) osvschema.Ran
 func sortBadSemver(a, b string) int {
```

Comment:
Hmm maybe rename to sortSemverLike? I'm guessing this also sorts good semver versions